stubs/Pygments/pygments/formatters/__init__.pyi

```diff
@@ -20,6 +20,7 @@ from .terminal import TerminalFormatter as TerminalFormatter
 from .terminal256 import Terminal256Formatter as Terminal256Formatter, TerminalTrueColorFormatter as TerminalTrueColorFormatter
 
 def get_all_formatters() -> Generator[type[Formatter[Any]], None, None]: ...
-def get_formatter_by_name(_alias, **options): ...
-def load_formatter_from_file(filename, formattername: str = "CustomFormatter", **options): ...
-def get_formatter_for_filename(fn, **options): ...
+def find_formatter_class(alias: str) -> type[Formatter[Any]]: ...
```

Comment:
In this case, `Formatter[Any]` doesn't need an explanation, as it's obvious why it's used.